cmd/transportRequestUploadCTS.go

```diff
@@ -91,5 +94,15 @@ func runTransportRequestUploadCTS(
 	action.WithConfigFile(config.DeployConfigFile)
 	action.WithDeployUser(config.OsDeployUser)
 
-	return action.Perform(cmd)
+	commonPipelineEnvironment.custom.transportRequestID = config.TransportRequestID
```

Comment:
Why store it in the CPE if it is anyway available in the config?

Reply:
Storing in the cpe makes it available for further steps.